anyway/views/news_flash/api.py

```diff
@@ -281,3 +282,34 @@ def normalize_query_param(key, value: list):
 def normalize_query(params: dict):
     params_non_flat = params.to_dict(flat=False)
     return {k: normalize_query_param(k, v) for k, v in params_non_flat.items()}
+
+
+@roles_accepted(BE_CONST.Roles2Names.Authenticated.value)
```

Comment:
Regarding the role, I believe we will need to have an additional role that allows to update data. We do not have the exact requirement yet. You can either add such a role, or use `Or_yarok` in the meantime.

Reply:
done